pocs/focuser/birger.py

```diff
@@ -112,7 +130,6 @@ def move_to(self, position):
         Does not do any checking of the requested position but will warn if the lens reports hitting a stop.
         Returns the actual position moved to in lens encoder units.
         """
-        self.logger.debug('Moving focus to {} encoder units'.format(position))
```

Comment:
What I was trying (and failing) to ask was whether there was really a need for a logging level below `DEBUG` to be available, i.e. does reading the logs at `INFO` level & above give you enough information for normal use?